docs/sql/README.md

```diff
@@ -0,0 +1,74 @@
+# SQL Queries in Starcounter
+
+## Data Manipulation Language (DML) Queries
+
+Starcounter 3.0 has inherited SQL Query Processor from Starcounter 2.3.2. Please refere to the (original documentation)[https://docs.starcounter.io/v/2.3.2/guides/sql] for the full specification.
+
+## Data Definition Language (DDL) Queries
+
+As of right now the following DDL statements are supported:
+
+### Create / drop database index
+
+```sql
+CREATE INDEX IX_IndexName ON TableName (ColumnName)
+```
+
+```sql
+DROP INDEX IX_IndexName ON TableName
+```
+
+### Create / drop database table
+
+```sql
+CREATE TABLE TableName
+(
+    BooleanColumn boolean,
```

Comment:
```suggestion
	BooleanColumn boolean,
```

Reply:
I did the other way around, changed the rest of the tabs to spaces.